mscore/startcenter.cpp

```diff
@@ -164,6 +164,17 @@ void Startcenter::readSettings()
       MuseScore::restoreGeometry(this);
       }
 
+//---------------------------------------------------------
+//   keyPressEvent
```

Comment:
keyPressEvent or keyReleaseEvent? Why didn't you implement keyPressEvent?

Reply:
I think onle the keyReleaseEvent is needed here? But the comment needs to get fix...

Reply:
Sorry, my mistake, Done now!